chrome/script.browserify.js

```diff
@@ -232,6 +236,47 @@ function getFaviconUrl(domain) {
   return null;
 }
 
+function launchURL() {
+  var what = this.what;
+  var entry = this.entry;
+  chrome.runtime.sendNativeMessage(
+    app,
+    { action: "get", entry: this.entry },
+    function(response) {
+      if (chrome.runtime.lastError) {
+        error = chrome.runtime.lastError.message;
+        m.redraw();
```

Comment:
nitpick: could you please reduce the nesting? 
It's a little difficult to see which "else" belongs to which "if" 🙂

```
if (error) {
  ...
  return; 
}
```

Reply:
Sure.